yt/yt/server/lib/job_proxy/config.h

```diff
@@ -156,6 +156,7 @@ class TJobProxyInternalConfig
 public:
     // Job-specific parameters.
     int SlotIndex = -1;
+    int ShardingKeyLength;
```

Comment:
Is it used?

Reply:
It's used for creating logs path for Job Proxy.